internal/cmd/argus/scrape-config/update/update.go

```diff
@@ -0,0 +1,130 @@
+package update
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/client"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+)
+
+const (
+	jobNameArg = "JOB_NAME"
+
+	instanceIdFlag = "instance-id"
+	payloadFlag    = "payload"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	JobName    string
+	InstanceId string
+	Payload    argus.UpdateScrapeConfigPayload
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("update %s", jobNameArg),
+		Short: "Updates a scrape configuration of an Argus instance",
+		Long: fmt.Sprintf("%s\n%s\n%s",
+			"Updates a scrape configuration of an Argus instance.",
+			"The payload can be provided as a JSON string or a file path prefixed with \"@\".",
+			"See https://docs.api.stackit.cloud/documentation/argus/version/v1#tag/scrape-config/operation/v1_projects_instances_scrapeconfigs_partial_update for information regarding the payload structure.",
```

Comment:
I think you meant
```suggestion
			"See https://docs.api.stackit.cloud/documentation/argus/version/v1#tag/scrape-config/operation/v1_projects_instances_scrapeconfigs_update for information regarding the payload structure.",
```
right?